index/miner/module/meta.go

```diff
@@ -18,9 +18,9 @@ import (
 )
 
 var (
```

Comment:
Metadata information got the largest cut. At this moment, this information has 0 value for FFS miner selection, keep  these updates contribute as tiny load as possible for now.